proto/destination.proto

```diff
@@ -236,3 +242,8 @@ message HttpStatusRange {
   uint32 max = 2;
 }
 
+message Backend {
+  string service = 1;
```

Comment:
What is stored in this value? Is it a kubernetes service name? is it a new authority value? If the latter, it should not be called service.

Reply:
Similarly, the semantics of weight values should be documented.